index.html

```diff
@@ -12,7 +12,7 @@
     <header class="menu-line">
         <div>
             <nav>
-                <a href="!" class="menu-box">
+                <a href="!" class="menu-box flex">
                     <img class="menu-icon" src="images/menu.png">
```

Comment:
- Kindly, adjust the space above `hamburger` as the previous code reviewer suggested, I think you are using the wrong image for your `nav`, use only the hamburger icon instead of using an image for your `nav` which will help you later 👍